lib/config/index.ts

```diff
@@ -67,6 +68,10 @@ export async function parseConfigs(
     config = mergeChildConfig(config, { force: { ...cliConfig } });
   }
 
+  if (!config.privateKey && config.privateKeyPath) {
+    config.privateKey = fs.readFileSync(config.privateKeyPath);
```

Comment:
```suggestion
    config.privateKey = await fs.readFile(config.privateKeyPath);
```

Should it be read with utf8? I'm not sure.

Reply:
In case `privateKey` should always contain the same kind of string I'd say yes. Otherwise the raw Buffer is returned when reading from file and passed directly to crypto's decrypt method.

Reply:
OK, if it works for you locally

Reply:
`readFile` requires a Callback and does not return a Promise. Need to revert the code suggestion

Reply:
No, import `fs-extra` instead

Reply:
see comment above, you need to use our fs wrapper